src/buggy_code.py

```
@@ -0,0 +1,56 @@
+    
+    def authenticate_user(self,pwd):
+        if self.Password==pwd:
+            return True
+        else:
+            return False
+    
+    def calculate_age(self, birth_date):
+        try:
+            age = datetime.now().year - birth_date.year
+            return age
+        except:
```

Ruff

Do not use bare `except`

(E722)